src/Impostor.sln

```diff
@@ -163,4 +163,4 @@ Global
 		{4DB56ADD-6D3D-4D0E-A047-9D7E7D40EF99} = {56DD9707-D811-4056-9E2C-8A9CC2479B07}
 		{70F97BB7-12A1-4C7A-B2A5-B962D14B813E} = {36AA9913-E6EA-4A6C-90E6-2FD3CC2E3124}
 	EndGlobalSection
-EndGlobal
+EndGlobal
```

Comment:
IDK WHAT THE FUCK HAPPENS HERE

Reply:
Your formatter is removing the final line of the file